mediapipe/tasks/c/components/containers/BUILD

```diff
@@ -209,3 +236,28 @@ cc_test(
         "@com_google_googletest//:gtest_main",
     ],
 )
+
+cc_library(
```

Comment:
I noticed this during import, but since this is not a shared utility this should live in the task's folder. I am moving this.

I will also move the lanugage_detector_result_converter.